src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -403,11 +403,18 @@ private Collection<FunctionDescriptor> ReadFunctions(ScriptHostConfiguration con
                         continue;
                     }
 
+                    if (functions.Count == ScriptConfig.MaximumFunctionCount)
+                    {
+                        AddFunctionError(functionName, string.Format(CultureInfo.InvariantCulture,
+                            "Maximun number of functions ({0}) exceeded. Ignoring function.", ScriptConfig.MaximumFunctionCount));
+                        continue;
```

Comment:
Yes, it should be `break`, unless we're going to display the name of all the skipped functions (which is probably unnecessary).